connector_ecommerce/data/ecommerce_data.xml

```diff
@@ -39,39 +39,44 @@
             <field name="standard_price">0.0</field>
             <field name="type">service</field>
             <field name="name">Discount Coupon</field>
-            <field name="categ_id" ref="product_categ_services"/>
+            <field name="categ_id" ref="product_categ_services" />
         </record>
 
         <record id="excep_parent_order_need_cancel" model="sale.exception">
-            <field name="name">Parent sales order canceled on the backend.</field>
-            <field name="description">The parent sales order has been canceled on the backend.
-You will not be able to process this sales order until the parent's one is canceled.</field>
+            <field name="name">Parent sales order canceled on the
+                backend.</field>
+            <field name="description">The parent sales order has been canceled
+                on the backend.
+                You will not be able to process this sales order until the parent's one
+                is canceled.</field>
             <field name="sequence">30</field>
             <field name="model">sale.order</field>
             <field name="code">if order.parent_need_cancel:
-    failed = True</field>
-            <field name="active" eval="True"/>
+                failed = True</field>
```

Comment:
That limitation only applies to Python files, not XML ones. As said, better to put it in one line, or the readibility inside Odoo will be bad.